File: JavaScript/프로그래머스/0/120837. 개미 군단/개미 군단.js
const solution = hp => {
    
    let ants = 0;
    
    // 장군개미 사용 (공격력 5)
    ants += Math.floor(hp / 5);
    hp %= 5;
    
    // 병정개미 사용 (공격력 3)
    ants += Math.floor(hp / 3);
    hp %= 3;
    
    // 남은 체력은 일개미가 처리 (공격력 1)
    ants += hp;

    return ants;
}

File: Java/프로그래머스/0/120837. 개미 군단/개미 군단.java
class Solution {
    public int solution(int hp) {
        
        int answer = 0;
        
        /** 장군개미 */
        answer += hp / 5;
        hp = hp % 5;
        
        /** 병정개미 */
        answer += hp / 3;
        hp = hp % 3;
        
        /** 일개미 */
        answer += hp;
        
        return answer;
        
    }
}